app/models/async_transaction/central_mail/va4142_submit_transaction.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module AsyncTransaction
+  module CentralMail
+    class VA4142SubmitTransaction < AsyncTransaction::Base
```

Comment:
Are we sure that we need an async transaction for 4142? 526 has one because it kicks off the flow, and due to all the sub-systems EVSS has to hit, is much more likely to timeout or run into a non-retryable validation error. CentralMail is pretty reliable and we're just posting a PDF there rather than a complex form.

Reply:
1. Sidekiq retrying is not dependent on the job being tracked by a class that extends `AsyncTransaction`. We do want to monitor if the job fails for any reason but async transactions don't really help with that as you'd need to get on a production server and run a query. 
2. I agree it could be useful to poll but I'm not sure it's possible :) the job is kicked off by the 526 job so there's no way to pass the job id up to the front end for it to poll against. And I think not polling is fine, 4142 is basically the same as supporting uploaded medical documents. The user doesn't need to know that it's a separate process from 526. 
3. Again this isn't made easier by having an async transaction record. Why the job failed is available, and easier to access via Sentry or AWS logs, and resending exhausted jobs will take manual intervention in any case.

Moving 526 to a background job was not our first choice; consistent timeouts for that endpoint in particular from EVSS are the main reason it's in a job. The child tasks are also in jobs as they depend on it succeeding. However as they don't need to report their status, outside of logging failures, they're not backed by async transaction records.

Reply:
@kreek : async transaction for 4142 has been removed